README.md

```diff
@@ -1,4 +1,97 @@
-# FLOCX Marketplace
-Just a simple Flask application for FLOCX Marketplace project.
-[More details on FLOCX](https://github.com/CCI-MOC/flocx)
+# flocx-market
 
+flocx-market is an OpenStack service for acting as a FLOCX marketplace to communicate with a FLOCX provider. [More details on FLOCX](https://github.com/CCI-MOC/flocx)
+
+### Service catalog
+#### Create the services
+
+```
+openstack service create --name flocx-market --description flocx-market marketplace
+```
+
+#### Register the services at an endpoint
+
+```
+openstack endpoint create flocx-market public http://example.com:XXXX
+```
+
+#### Create service users
+
+Create a project for the service users:
+
+```
+openstack project create service --domain default
+```
+
+Create service users for the relevant services:
+
+```
+openstack user create flocx-market --password PASSWORD
+```
+
+Assign the admin role to the user-project pair:
+
+```
+openstack role add --project service --user flocx-market admin
+```
```

Comment:
Let's move this entire section running openstack commands to the end - after instructions on how to start up the API.